### PR TITLE
Update Static Tools Logo and CodeQL doc

### PR DESCRIPTION
**Pull request by lauragonzalezzz**
Includes a majority refactor, updated list of queries, as well as a new navigation section.

### CODE REVIEW

**Comment by DOMARS** [comment]
Let me know if I can help with any of the doc system URL warnings.

**Comment by lauragonzalezzz** [comment]
@DOMARS Thank you! Do you happen to know the relative links to these files?

- https://learn.microsoft.com/en-us/windows-hardware/test/hlk/
- https://learn.microsoft.com/en-us/windows-hardware/design/compatibility/whcp-certification-process

**Comment by DOMARS** [comment]
@lauragonzalezzz - I think for links on learn - you can just say `/windows-hardware/design/compatibility/whcp-certification-process`

**Comment by lauragonzalezzz** [comment]
@DOMARS Can I get this merged?

**Comment by DOMARS** [comment]
@lauragonzalezzz - Sure will do that right now.

File: windows-driver-docs-pr/devtest/static-tools-and-codeql.md [hunk 1]
```diff
@@ -82,241 +116,199 @@ We will use the [CodeQL command line tools (CLI)](https://codeql.github.com/docs
...
+   CALL devenv /Edit C:\codeql-home\databases\kmdf.sarif
+   SET ERRORLEVEL = 0
+   ```
+
+1. The [devenv.exe / Edit](/visualstudio/ide/reference/edit-devenv-exe) option is used in the batch file to open the SARIF results file in the existing instance of Visual Studio. To view the SARIF results install the [Microsoft SARIF Viewer for Visual Studio](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer) and refer to the instructions there for more information.
 
-You can specify a timeout for the entire operation with the *"–timeout=[seconds]"* flag. This can be useful for analysis on queries without being limited by a single, long-running query. More options to tweak analysis optimizations are described in [database analyze](https://codeql.github.com/docs/codeql-cli/analyzing-databases-with-the-codeql-cli/).
+1. In the driver project, navigate to project properties. In the  **Configuration** pull down, select the build configuration that you wish to check with CodeQL, we recommend "Release. Creating the CodeQL database and running the queries takes a few minutes, so we don't recommend you run CodeQL on the Debug configuration of your project.
```
Comment (NateD-MSFT) [inline windows-driver-docs-pr/devtest/static-tools-and-codeql.md:282]: I think what's trying to be said here is essentially, "CodeQL can take a lot of time, don't run it on every dev build as you code", correct? I wonder if there's a more natural way to say this than the Release/Debug divide.

Comment (NateD-MSFT) [inline windows-driver-docs-pr/devtest/static-tools-and-codeql.md:282]:
```suggestion
1. In the driver project, navigate to project properties. In the  **Configuration** pull down, select the build configuration that you wish to check with CodeQL, we recommend "Release". Creating the CodeQL database and running the queries takes a few minutes, so we don't recommend you run CodeQL on the Debug configuration of your project.
```

File: windows-driver-docs-pr/devtest/static-tools-and-codeql.md [hunk 2]
```diff
@@ -82,241 +116,199 @@ We will use the [CodeQL command line tools (CLI)](https://codeql.github.com/docs
...
+   1>[3/4 eval 4.5s] Evaluation done; writing results to codeql-cpp\Likely Bugs\Underspecified Functions\ImplicitFunctionDeclaration.bqrs.
+   1>[4/4 eval 5.2s] Evaluation done; writing results to codeql-cpp\Likely Bugs\Underspecified Functions\MistypedFunctionArguments.bqrs.
+   1>Shutting down query evaluator.
+   1>Interpreting results.
+   1>">>> Loading SARIF Results in Visual Studio <<<"
+   ```
 
 ## Troubleshooting
 
+> If you are certifying with WHCP, the most common issue is version mismatch between HLK version and Windows Release version. Please ensure you are using the HLK version associated with the Windows release you are targeting, the associated branch in the Windows Driver Developer Supplemental Tools repository, and the subsequent CodeQL CLI version. For HLK/Windows Release compatability matrix, see [Windows Hardware Lab Kit](/windows-hardware/test/hlk/) and for Windows Release/Windows Driver Developer Supplemental Tools repo branch/CodeQL CLI version, see the WHCP table in the [CodeQL Setup](#1-codeql-setup) section.
```
Comment (NateD-MSFT) [inline windows-driver-docs-pr/devtest/static-tools-and-codeql.md:308]:
```suggestion
> If you are certifying with WHCP, the most common issue is version mismatch between HLK version and Windows Release version. Please ensure you are using the HLK version associated with the Windows release you are targeting, the associated branch in the Windows Driver Developer Supplemental Tools repository, and the correct CodeQL CLI version for that branch. For HLK/Windows Release compatability matrix, see [Windows Hardware Lab Kit](/windows-hardware/test/hlk/) and for Windows Release/Windows Driver Developer Supplemental Tools repo branch/CodeQL CLI version, see the WHCP table in the [CodeQL Setup](#1-codeql-setup) section.
```

**Pull request merged**